src/FormulaParser.elm

```diff
@@ -0,0 +1,182 @@
+module FormulaParser exposing (parse, parseSigned, parseTerm)
```

Comment:
Make this Formula.Parser (i.e. `src/Formula/Parser.elm`), similar for `Formula.Signed`